### DISCUSSION TITLE
Best hardware and SSDs setup for optimal performance with kvrocks

### DISCUSSION THREAD

**Discussion by adulau** [apache/kvrocks#1777] [General]
What would be the best approach to select hardware and SSDs in order to have optimal performance with kvrocks? Any specific recommendations that users should consider.

**Comment by git-hulk** [comment]
I know less about choosing the hardware. From my personal perspective, disk IOPS and bandwidth are the main factors to affect the performance.

**Reply by bbqccx** [reply]
In my scenario：
p4510： 128K sequential read/write MB/s is 3200/3000； 4K random read/write iops is 637K/139K    
p5510:    128K sequential read/write MB/s is 7000/4194； 4K random read/write iops is 930k/190k
with  rocksdb.block_size 16KB ，in 128-byte random set scenario ：
p4510 (pcie 3.0) can stable at QPS 170K，p5510 (pcie 4.0) can stable at QPS 220K
but please note, when two kvrocks instances on one nvme-ssd，the stable value of QPS will decrease

kvrokcs 2.4 and The main parameters are as follows：
work 16
max_background_compactions 8
max_background_flushes 8
max_sub_compactions 3
compression lz4
max_write_buffer_number 10
level0_slowdown_writes_trigger 40
level0_stop_writes_trigge 64
